frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -69,16 +88,9 @@ class Recipe extends Component {
             <Header user={user} setJwt={setJwt} />
           </div>
         </div>
-        <div>
-          <button className="btn navbar-btn btn-lg edit-button" type="button">
-            <img src="https://img.icons8.com/windows/32/ffa616/edit.png" alt="edit" />
-            <p id="edit-icon-text">Edit</p>
-          </button>
-          <button className="btn navbar-btn btn-lg cancel-button" type="button">
-            <img src="https://img.icons8.com/windows/32/ffa616/cancel.png" alt="delete" />
-            <p id="delete-icon-text">Delete</p>
-          </button>
-        </div>
+        <span>
+          { user ? renderButtons() : 'null' }
```

Comment:
I don't think you need the `span` tag. I think you also want `null` without the single quotes here, since you don't want to return the string "null".